src/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import 'assets/styles/tailwind.css';
```

Comment:
Hmmm, we have this style import here, and we have another on inside `src/App.js`. Could we add them all here?